debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -377,6 +377,52 @@ public static FieldNameAdjustmentMode parse(String value) {
         }
     }
 
+    /**
+     * The set of predefined SourceLocation options
+     */
+    public enum SourceLocation implements EnumeratedValue {
```

Comment:
Please move this to the `CloudEventsConverterConfig` class.  The `CommonConnectorConfig` should only contain things that are specific to actual connectors while this is specific to CloudEvents only.

Reply:
done